src/apps/console/routes/_main+/$account+/packages+/access-management/cred-resources.tsx

```diff
@@ -77,113 +74,100 @@ const ExtraButton = ({ onDelete }: { onDelete: () => void }) => {
 };
 
 const RegistryUrlView = () => {
-  const { copy } = useClipboard({
-    onSuccess() {
-      toast.success('Registry url copied successfully.');
-    },
-  });
+  const { copy } = useClipboard({});
```

Comment:
Instead of creating states and designs again and again in every place. better we can create components and use them everywhere.

Reply:
done